browser-test/src/support/applicant_questions.ts

```diff
@@ -191,7 +191,7 @@ export class ApplicantQuestions {
   }
 
   async clickNext() {
-    await this.page.click('text="Next"')
+    await this.page.click('text="Save And Next"')
```

Comment:
Are you working with Elise on translations for this?

Reply:
Yes.

Reply:
Yes.